docs/tutorial/nexus-to-nomad.md

```diff
@@ -0,0 +1,61 @@
+# Uploading NeXus files to NOMAD
+
+Great choice! [Nomad](https://nomad-lab.eu/nomad-lab/tutorials.html) makes it easier than ever to work with your research data. At this point you are probably have an idea of what FAIR data is, [wilkinson et al](...). Even if you don't, it doesn't matter. Nomad provides a simple graphical interface that let's you collect and have your data ready for publication.
+
+In this tutorial, we will go through how one can upload their NeXus files to Nomad.
+
+Nomad, as a FAIR data platform, supports NeXus and allows users to upload their NeXus (.nxs) files directly. These files get interpreted and added to your Nomad account with complete control on how you would like to present and publish them alongside your research.
+
+## Create an account
+
+The very first thing you need to do is get a Nomad account if you don't have one. **!!!You can go here and click this then that**
+
+1. Navigate to [nomad-lab.eu](https://nomad-lab.eu/prod/v1/gui/about/information)
+2. Click on ```Login / Register``` on the top right corner.
+
+<img src="media/login-register.gif" />
+
+## Create an Upload
+
+Nomad allows you to have a draft working space called an **upload**. This allows you to test and prepare how your data will look in Nomad before you publish it.
+
+Go to ```Publish -> Uploads```
+
+<img src="media/uploads.gif" />
+
+<br />
+
+Click ```Create a new upload```
+
+<img src="media/create-new-upload.gif" />
+
+## Upload your NeXus file
```

Comment:
Shall we add other process where you upload your original file and choose the NeXusDataConverter?